charts/latest/azurefile-csi-driver/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: latest
```

Comment:
after second thought, I think we should keep `latest` tag, `latest` means current master branch version, and if we specify `v0.8.0`, we need to change this version num every time there is new release.

Reply:
sorry for the turn around, while I think using `latest` should be more appropriate for maintenance in the future.

Reply:
Hi, @andyzhangx so we have a few other problems while using `latest` as the version.
Helm charts specify that the chart version has to be of SemVer2 format.
https://helm.sh/docs/topics/charts/#charts-and-versioning

This is needed to parse the version while using `--version` tag with `helm install`. Now the keyword `latest` does not follow this rule. Hence I am getting the following error:
```
Error: chart "azurefile-csi-driver" matching version "latest" not found in azurefile-csi-driver index. (try 'helm repo update'). improper constraint: latest
```
 When I try running
 ```
helm install --name azurefile-csi-driver azurefile-csi-driver/azurefile-csi-driver --namespace kube-system --version latest --debug
```
I confirmed this problem by checking these issues:
* https://github.com/helm/helm/issues/4109
* https://github.com/helm/helm/issues/3600

AFAIK there is also no way to specify a chart as default in `index.yaml`. What helm does is sort the chart versions and install the latest one if the `version` is not explicitly mentioned. 
https://github.com/helm/helm/blob/15d9a6190f5e3ed01b5f2f670be48e7a0290518d/pkg/repo/index.go#L152

Let me know how you would like me to move forward from here.

Reply:
hi, what about remove `latest` in helm installation, so the biggest version in helm is `v0.7.0` , and keep charts\latest unchanged
CI test would run on chart `latest` , and don't change deploy version, keep it as `latest` ?

Reply:
Hi, so you don't want me to include `latest` inside `index.yaml`? In this case, the default installation will be `v0.7.0`.  If that's the case. I would again suggest following what elastic does. https://github.com/elastic/helm-charts/tree/master/elasticsearch#installing. Include stable version installation via helm repo and development(latest) version installation directly.